src/Engine/JN_TTF/JN_TTF.cpp

```diff
@@ -0,0 +1,23 @@
+#include "JN_TTF.h"
```

Comment:
This should be JN_Ttf, JN_TTF looks like a constant.

Reply:
But TTF is like SDL, it's a capcase thing... Would you still want it Ttf?